<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="number1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="number2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNumbers">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subNumbers">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mulNumbers">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiply</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="answer">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>526</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
